### PR TITLE
got a basic landing page looking good 

### PR DESCRIPTION
**Pull request by LFMB**
It's time to link form to MySQL database

### CODE REVIEW

**Comment by erin-doyle** [comment]
Hey @LFMB looks like you've got some conflicts to resolve.  You should rebase your branch a lot has changed probably since you branched

File: react-ui/package.json [hunk 1]
```diff
@@ -1,20 +1,30 @@
...
+    "node-sass": "^4.13.1",
+    "postcss-loader": "^3.0.0",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
     "react-scripts": "3.4.1",
     "serve": "^11.3.0"
   },
   "scripts": {
     "start": "react-scripts start",
+    "start-api": "source venv/bin/activate && python app.py",
```
Comment (erin-doyle) [inline react-ui/package.json]: This and `test-api` have been removed for instance

File: react-ui/src/App.js [hunk 2]
```diff
@@ -1,13 +1,27 @@
...
 
 
 function App() {
+  const [currentTime, setCurrentTime] = useState(0);
+
+  useEffect(() => {
+    fetch('/time').then(res=>res.json()).then(data=>{
+      setCurrentTime(data.time); 
+    })
+  },[]);
```
Comment (erin-doyle) [inline react-ui/src/App.js]: This test stuff has been removed now, we can leave it out.

File: react-ui/src/components/splash-banner.jsx [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
+import logo from '../assets/logo-colored-horizontal.png';
+
+
+
+
+function HomeUIWrapper() {
+  return (
+    <div className="home-ui-wrapper">
+      <header className="splash-header">
+        <img src={logo} className="logo" alt="Nainers Matter logo" />
```
Comment (erin-doyle) [inline react-ui/src/components/splash-banner.jsx]: Nainers? 😝

Comment (LFMB) [reply react-ui/src/components/splash-banner.jsx]: the component isn't being used at the moment - so meh.

Good eye though!!

**Pull request merged**